examples/json/json.go

```diff
@@ -1,119 +1,122 @@
-
-    // Now let's look at decoding JSON data into Go
-    // values. Here's an example for a generic data
-    // structure.
-    byt := []byte(`{"num":6.13,"strs":["a","b"]}`)
-
-    // We need to provide a variable where the JSON
-    // package can put the decoded data. This
-    // `map[string]interface{}` will hold a map of strings
-    // to arbitrary data types.
-    var dat map[string]interface{}
-
-    // Here's the actual decoding, and a check for
-    // associated errors.
-    if err := json.Unmarshal(byt, &dat); err != nil {
-        panic(err)
-    }
-    fmt.Println(dat)
-
-    // In order to use the values in the decoded map,
-    // we'll need to cast them to their appropriate type.
-    // For example here we cast the value in `num` to
-    // the expected `float64` type.
-    num := dat["num"].(float64)
-    fmt.Println(num)
-
-    // Accessing nested data requires a series of
-    // casts.
-    strs := dat["strs"].([]interface{})
-    str1 := strs[0].(string)
-    fmt.Println(str1)
-
-    // We can also decode JSON into custom data types.
-    // This has the advantages of adding additional
-    // type-safety to our programs and eliminating the
-    // need for type assertions when accessing the decoded
-    // data.
-    str := `{"page": 1, "fruits": ["apple", "peach"]}`
-    res := Response2{}
-    json.Unmarshal([]byte(str), &res)
-    fmt.Println(res)
-    fmt.Println(res.Fruits[0])
-
-    // In the examples above we always used bytes and
-    // strings as intermediates between the data and
-    // JSON representation on standard out. We can also
-    // stream JSON encodings directly to `os.Writer`s like
-    // `os.Stdout` or even HTTP response bodies.
-    enc := json.NewEncoder(os.Stdout)
-    d := map[string]int{"apple": 5, "lettuce": 7}
-    enc.Encode(d)
+	// Come prima cosa vedremo come fare l'encoding di
+	// datatype basici per ottenere stringhe JSON. Ecco
+	// degli esempi per dei valori atomici.
+	bolB, _ := json.Marshal(true)
+	fmt.Println(string(bolB))
+
+	intB, _ := json.Marshal(1)
+	fmt.Println(string(intB))
+
+	fltB, _ := json.Marshal(2.34)
+	fmt.Println(string(fltB))
+
+	strB, _ := json.Marshal("gopher")
+	fmt.Println(string(strB))
+
+	// E ora degli esempi con le slice e le map, che
+	// ritornano degli array e oggetti JSON come ti
+	// aspetteresti.
+	slcD := []string{"mela", "pesca", "pera"}
+	slcB, _ := json.Marshal(slcD)
+	fmt.Println(string(slcB))
+
+	mapD := map[string]int{"mela": 5, "lattuga": 7}
+	mapB, _ := json.Marshal(mapD)
+	fmt.Println(string(mapB))
+
+	// Il package JSON può encoddare automaticamente
+	// dei data type personalizzati, ad esempio delle
+	// struct. Includerà solo i field esportati
+	// nell'output encoddato e di default userà i nomi
+	// dei field per le chiavi JSON.
+	res1D := &Response1{
+		Pagina: 1,
+		Frutti: []string{"mela", "pesca", "pera"}}
+	res1B, _ := json.Marshal(res1D)
+	fmt.Println(string(res1B))
+
+	// Puoi usare dei "tag" sulle dichiarazioni dei field
```

Comment:
Qui ci andrebbe il markdown del corsivo no? (intendo intorno a tag). 
In realtà anche le virgolette possono andare, vedi tu